apps/omg_performance/test/omg_performance/byzantine_event_test.exs

```diff
@@ -83,6 +113,144 @@ defmodule OMG.Performance.ByzantineEventsTest do
     assert Enum.count(ByzantineEvents.get_byzantine_events("invalid_exit")) > 10
   end
 
+  @tag fixtures: [:perf_test, :mix_based_child_chain, :mix_based_watcher]
```

Comment:
these tests bellow need a bit more data for me ... I can't digest them because I don't know what the plan is and what they test.

Reply:
falls under an earlier comment, let's agree on how to describe the tests there and I'll update all.